ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -227,7 +238,7 @@ export default function CreatePassword({
             titleText={t('newPassword')}
             value={password}
             titleDetail={
-              <Typography variant={TypographyVariant.H7}>
+              <Text variant={TextVariant.bodyXs}>
```

Comment:
Let's replace the Text and link element  with the ButtonLink component. Then we will be reducing the html bloat and using the correct semantic html element
```
 <ButtonLink
    variant={TextVariant.bodySm}
    data-testid="show-password"
    className="create-password__form--password-button"
    onClick={(e) => {
      e.preventDefault();
      setShowPassword(!showPassword);
    }}
    marginBottom={1}
  >
    {showPassword ? t('hide') : t('show')}
  </ButtonLink>
```

Reply:
Sure, 
Will do it now.